loom/script/compiler/lsTypeVisitor.h

```diff
@@ -854,7 +854,7 @@ class TypeVisitor : public TraversalVisitor {
                     }
                 }
 
-                if (tinfo->isTemplate())
+                if (tinfo && tinfo->isTemplate())
```

Comment:
Oh right, before template info wasn't a pointer in memberInfo, but a.. field on the stack? Whatever you call it. So getTemplateInfo never returned NULL.